HIP/hip-632.md

```diff
@@ -10,65 +10,118 @@ status: Accepted
 last-call-date-time: 2023-01-10T07:00:00Z
 created: 2022-11-28
 discussions-to: https://github.com/hashgraph/hedera-improvement-proposal/discussions/638
-updated: 2022-12-13, 2022-12-20, 2023-01-03, 2023-01-18
-requires: 631
+updated: 2024-07-15
+requires: 583
 ---
 
 ## Abstract
 
-Virtual Addresses introduced a list of ECDSA public addresses that are stored in the account entity and serve as the account identifier on the EVM for transactions. This is in addition to the `Hedera Account ID` that is used by default when no virtual addresses exist.
+Hedera accounts have the capability to have a linked _EVM address alias_, an EVM address.  In this way smart contracts can deal
```

Comment:
perhaps it'll be helpful to link/refer to the table in HIP 583 that defines the various address formats.